docs/Earn/switch/switch-introduction.md

```diff
@@ -1,6 +1,48 @@
 ---
-title: Itroduction
+title: Ankr Switch
 id: switch-introduction
 ---
 
-Coming soon...
\ No newline at end of file
+# Ankr Switch
+
+## What is Ankr Switch?
+
+Ankr Switch is a product for switching between pairs of rewards earning and reward bearing tokens.
+
+### What are reward bearing and rewards earning tokens?
+
+They are tokens issued to you by Ankr when staking in Arnk Earn, in 1:1 ratio to your staked assets.
```

Comment:
when staking with **Ankr Earn**. These tokens are in a 1:1 ratio to...

Reply:
thx